TechnicalDocuments/TechnicalDebt.md

```diff
@@ -0,0 +1,115 @@
+
+| View Model | Comments |
+| ---------- | -------- |
+| AppointmentCancellationViewModel | |
+| AppointmentConsultationNotesViewModel | |
+| AppointmentDetailsViewModel | |
+| AppointmentNotesViewModel | |
+| AppointmentPrescriptionViewModel | CE-30 |
+| AppointmentReferralViewModel | |
+| AppointmentReferralsViewModel | |
+| AppointmentReplayViewModel | |
+| AppointmentListViewModel | |
+| BookAppointmentViewModel | |
+| PractitionerDetailsViewModel | |
+| ChatHistoryViewModel | |
+| MedicalHistoryViewModel | |
+| PersonalDetailsViewModel | |
+| InfoItemsViewModel | |
+| GPDetailsViewModel | |
+| AddAddressViewModel | |
+| PrivacySettingsViewModel | |
+| ChooseCountryViewModel | |
+| ForgotPasswordViewModel | |
+| NotificationsViewModel | |
+| ProfileViewModel | |
+| SignInViewModel | |
+| NHSRegistrationStep1ViewController | |
+| NHSRegistrationStep2ViewController | |
+
+## Accessing Business Controllers Directly instead of using the SDK.
+
+Some of these business controllers are not defined inside the SDK. Most of them will be moved to the SDK, but some of them will stay in the Babylon project. Once the first, complete version of the SDK has been published there should not be any business controllers left in BabylonCore. Business controllers that are defined outside the SDK should have a documenting comment that explain why they are not part of the SDK.
+
+| Business Controller | Accessed From |
+| ------------------- | ------------- |
+| AppointmentBusinessControllerProtocol | BabylonAppointmentsUI, Babylon |
+| AddressBusinessControllerProtocol | BabylonAppointmentsUI, BabylonUI, Babylon |
+| PractitionerBusinessController | BabylonAppointmentsUI |
+| PatientBusinessControllerProtocol | BabylonAppointmentsUI, BabylonClinicalRecordsUI, BabylonUI, Babylon |
+| PrescriptionBusinessControllerProtocol | BabylonAppointmentsUI, Babylon |
+| PharmaciesBusinessControllerProtocol | BabylonAppointmentsUI, BabylonClinicalRecordsUI |
+| ImageBusinessControllerProtocol | BabylonAppointmentsUI, BabylonClinicalRecordsUI, BabylonMonitor, Babylon |
+| BookAppointmentBusinessControllerProtocol | BabylonAppointmentsUI |
+| PrivacyBusinessControllerProtocol | BabylonAppointmentsUI, BabylonChatBotUI, BabylonUI |
+| PublicHealthcareIdentifierBusinessController | BabylonAppointmentsUI, BabylonClinicalRecordsUI, BabylonUI, Babylon |
+| SearchAssistantBusinessController | BabylonChatBotUI |
+| MedicalHistoryBusinessController | BabylonClinicalRecordsUI |
+| GenderBusinessControllerProtocol | BabylonClinicalRecordsUI, BabylonUI, Babylon |
+| GeolocationBusinessController | BabylonClinicalRecordsUI, BabylonUI |
+| InfoItemsBusinessControllerProtocol | BabylonClinicalRecordsUI |
+| MapsBusinessControllerProtocol | BabylonClinicalRecordsUI |
+| GPDetailsBusinessController | BabylonClinicalRecordsUI |
+| HealthcheckReportBusinessControllerProtocol | BabylonMonitor, BabylonUI, Babylon |
```

Comment:
This likely won't become part of the SDK, since all the contents used inside the report web page is already available through the SDK anyway.